src/intel-map.coffee

```diff
@@ -10,8 +10,6 @@
 # Commands:
 #   hubot intelmap for <search>
 #
-# Author:
-#   therealklanni
```

Comment:
@therealklanni I was the original author of this code though. The problem with "author" names in scripts is exactly this. Would you prefer that I have replaced your name with mine? Should I have added mine below yours? Should I have also added the name of the person who split this code from the original hubot-ingress.coffee script?

See pull request https://github.com/hubot-scripts/hubot-ingress/pull/1